src/NServiceBus.Core.Tests/API/APIApprovals.ApproveNServiceBus.approved.txt

```diff
@@ -2683,6 +2702,7 @@ namespace NServiceBus.Unicast.Queuing
 }
 namespace NServiceBus.Unicast.Routing
 {
+    [System.ObsoleteAttribute("No longer used, safe to remove. Will be removed in version 7.0.0.", true)]
```

Comment:
The static router has been around but are no longer used. What are you suggesting?